src/lib/trajectory/parse.ts

```
@@ -836,498 +507,390 @@
+    if (is_ase_format(data, filename)) return parse_ase_trajectory(data, filename)
     if (is_torch_sim_hdf5(data, filename)) {
       return await parse_torch_sim_hdf5(data, filename)
     }
-
-    // If not a recognized HDF5 format, throw error
-    throw new Error(`Unsupported binary file format`)
+    throw new Error(`Unsupported binary format`)
   }
 
-  // Handle string data (raw file content)
   if (typeof data === `string`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)